packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/RecipientContainer.vue

```diff
@@ -35,7 +37,7 @@ export default {
     }
   },
 
-  data() {
+  data(): { formattedRecipient: Recipient } {
```

Comment:
is the `hasAvatar` really working/reactive? 🤔

Reply:
Well, before it was completely broken 

I think in another PR I had to move a property from data() to a Ref in setup and that fixed nested reactivity ... don't ask me why, we could do that here too - to be safe(r)

Do you have a reproducer to show it's *not* reactive?

In the updated snapshot you can at least see it's working at all now

Reply:
... but that's not better or worse than before, let's open a follow up ticket?